### PR TITLE
Dropping unused prometheus labels produced on google kubernetes clusters

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**

### CODE REVIEW

File: prometheus.yml [hunk 1]
```diff
@@ -141,6 +143,8 @@ scrape_configs:
         target_label: __metrics_path__
         replacement: /api/v1/nodes/${1}/proxy/metrics/cadvisor
         action: replace
+      - action: labeldrop
+        regex: (beta_kubernetes_io_arch|beta_kubernetes_io_os|kubernetes_io_arch|kubernetes_io_os|cloud_google_com_gke_max_pods_per_node|cloud_google_com_gke_os_distribution|cloud_google_com_gke_logging_variant|cloud_google_com_gke_container_runtime|cloud_google_com_private_node)
```
Comment (pwilczynskiclearcode) [inline prometheus.yml:147]: https://eu-metrics-monitoring.livepeer.live/grafana/goto/hsPFS1cSg?orgId=1

File: prometheus.yml [hunk 2]
```diff
@@ -64,6 +64,8 @@ scrape_configs:
         target_label: __metrics_path__
         replacement: /api/v1/nodes/${1}/proxy/metrics
         action: replace
+      - action: labeldrop
+        regex: (beta_kubernetes_io_arch|beta_kubernetes_io_os|kubernetes_io_arch|kubernetes_io_os|cloud_google_com_gke_max_pods_per_node|cloud_google_com_gke_os_distribution|cloud_google_com_gke_logging_variant|cloud_google_com_gke_container_runtime|cloud_google_com_private_node|cloud_google_com_gke_boot_disk|cloud_google_com_gke_cpu_scaling_level|cloud_google_com_gke_provisioning)
```
Comment (hjpotter92) [inline prometheus.yml:68]:
```regex
((beta_)?kubernetes_io_(arch|os)|cloud_google_com_gke_(max_pods_per_node|os_distribution|logging_variant|container_runtime|boot_disk|cpu_scaling_level|provisioning)|cloud_google_com_private_node)
```

Comment (pwilczynskiclearcode) [reply prometheus.yml:68]: Your version is nicely reduced but how about keeping it simple and verbose?

**Comment by pwilczynskiclearcode** [comment]
https://eu-metrics-monitoring.livepeer.live/grafana/goto/hsPFS1cSg?orgId=1 the labels don't get dropped after the deployment but I think it's not that important atm.

**Pull request merged**